languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,23 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ILS_VERBE_TOUS" name="ils verbe tous" default="temp_off">
```

Comment:
This rule produces too many false positives in the rule editor. Please see if those can be eliminated, otherwise the rule cannot be merged.

>Pour cela ils embarquent **tout un arsenal** électronique.